sway/tree/container.c

```diff
@@ -1423,7 +1423,7 @@ struct sway_container *container_split(struct sway_container *child,
 		}
 	}
 
-	struct sway_seat *seat = input_manager_get_default_seat();
+	struct sway_seat *seat = input_manager_current_seat();
```

Comment:
I don't see why container_split should use the default seat while the reset use the current seat, so I've changed it.